ui/app/abilities/variable.js

```diff
@@ -38,10 +39,48 @@ export default class Variable extends AbstractAbility {
   )
   canDestroy;
 
-  @computed('rulesForNamespace.@each.capabilities')
-  get policiesSupportVariableView() {
-    return this.rulesForNamespace.some((rules) => {
-      return get(rules, 'SecureVariables');
+  @computed('token.selfTokenPolicies')
+  get policiesSupportVariableList() {
```

Comment:
Why doesn't this support accepting a `path` as a parameter like the other policy checks in place?

Reply:
My feeling here is the best way for this to operate is to let the server (200 or 403/404) tell us what we need to know. As such, the `list variables` capability is only meant to be used for "do they have access to variables, generally" as far as the UI is concerned. No path needed for that.